lib/providers/buildings_data.dart

```diff
@@ -25,15 +22,13 @@ class BuildingsData extends ChangeNotifier {
 
   BuildingsData() {
     // Make one big set of buildings that has sgw + loy buildings
-    allBuildings = Search.supported.whereType<Building>().toList();
+    University.concordia.children.forEach((campus) {
+      allBuildings.addAll(campus.children.whereType<Building>());
+    });
```

Comment:
This implies that the provider can't be lazy loaded, since we load assets during the splash screen. If it is lazy loaded it would be instantiated before the splash screen is created resulting in there being no data.